crates/host/src/wasmbus/mod.rs

```diff
@@ -3578,6 +3686,49 @@ impl Host {
         Ok(res.into())
     }
 
+    #[instrument(skip(self))]
+    async fn handle_config_get_one(&self, entity_id: &str, key: &str) -> anyhow::Result<Bytes> {
+        trace!("getting config");
+        let json = match self
+            .config_data_cache
+            .read()
+            .await
+            .get(entity_id)
+            .and_then(|map| map.get(key))
+        {
+            Some(data) => json!({
```

Comment:
Can we re-use `GetConfigKeyResponse` here?

Reply:
I think we should use the types everywhere, but there are two main reasons I didn't:

1. We should have these types in a separate crate to be used by both the client and the server (we could put them in the host as well, but that is a heavy crate to pull in)
2. I was matching the style done elsewhere in this code to maintain consistency

Reply:
I'm okay with this for now, and agree we should move common types somewhere in the future 👍